lib/context-menu-handlers.js

```diff
@@ -126,6 +126,23 @@ function init(panel) {
     label: contextMenuLabel,
     context: [
       cm.URLContext(['*.vimeo.com']),
+      cm.SelectorContext('.iris_video-vital__overlay.iris_link-box')
+    ],
+    contentScript: `self.on('click', function (node, data) {
+      self.postMessage('https://vimeo.com' + node.href);
+    });`,
+    onMessage: (url)=> {
```

Comment:
The contentScript and `onMessage` callback here are duplicates of the one below. Is it possible to specify two contexts for a single cm.Item? If not, maybe assign to a variable to avoid duplication